### PR TITLE
Upgrade tempto version to 1.4

### PR DESCRIPTION
**Pull request by kokosing**
Upgrade tempto version to 1.4

### CODE REVIEW

File: presto-yarn-test/src/test/java/com/teradata/presto/yarn/test/slider/SliderStatusTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        SliderStatus sliderStatus = new SliderStatus(statusJson);
+
+        assertThat(sliderStatus.getLiveComponentsHost("UNKNOWN")).isEmpty();
+        assertThat(sliderStatus.getLiveContainers("UNKNOWN")).isEqualTo(0);
+
+        assertThat(sliderStatus.getLiveContainers("COORDINATOR")).isEqualTo(1);
+        assertThat(sliderStatus.getLiveComponentsHost("COORDINATOR")).containsExactly("kogut-vsphere-default-master");
+
+        assertThat(sliderStatus.getLiveComponentsHost("WORKER")).containsExactly(
+                "kogut-vsphere-default-slave2",
```
Comment (anusudarsan) [inline presto-yarn-test/src/test/java/com/teradata/presto/yarn/test/slider/SliderStatusTest.java:40]: this will fail in jenkins right?

**Comment by anusudarsan** [comment]
LGTM mod presto-td-yarn-product-tests-cdh passing

File: presto-yarn-test/src/test/java/com/teradata/presto/yarn/test/slider/SliderStatusTest.java (hunk 1 above)
Comment (kokosing) [inline presto-yarn-test/src/test/java/com/teradata/presto/yarn/test/slider/SliderStatusTest.java:40]: no, note this is an unit test, see that travis passed

File: presto-yarn-test/etc/docker/tempto-configuration-docker-local.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ssh:
+  identity: /workspace/etc/docker/key
```
Comment (anusudarsan) [inline presto-yarn-test/etc/docker/tempto-configuration-docker-local.yaml:2]: where is /workspace? Both here and in run_on_docker.sh?

File: presto-yarn-test/src/main/java/com/teradata/presto/yarn/test/PrestoClusterTest.java [hunk 3]
```diff
@@ -221,14 +227,15 @@ public void multiNodeWithPlacementCheckingConnectors()
...
-
     public void waitForNodesToBeActive(final QueryExecutor queryExecutor, final PrestoCluster prestoCluster)
     {
         retryUntil(() ->
         {
-            QueryResult result = queryExecutor.executeQuery("select count(*) from system.runtime.nodes");
-            log.debug("Number of active nodes: " + String.valueOf(result.rows()));
-            return result.rows().equals(singletonList(prestoCluster.getAllNodes().size()));
+            QueryResult result = queryExecutor.executeQuery("select * from system.runtime.nodes");
+            Collection<String> allNodes = prestoCluster.getAllNodes()   ;
```
Comment (anusudarsan) [inline presto-yarn-test/src/main/java/com/teradata/presto/yarn/test/PrestoClusterTest.java:234]: extra space before `;`

**Pull request merged**